stage1/gc/gc.go

```diff
@@ -43,13 +43,15 @@ const (
 )
 
 var (
-	debug bool
-	log   *rktlog.Logger
-	diag  *rktlog.Logger
+	debug       bool
+	log         *rktlog.Logger
+	diag        *rktlog.Logger
+	localConfig string
 )
 
 func init() {
 	flag.BoolVar(&debug, "debug", false, "Run in debug mode")
+	flag.StringVar(&localConfig, "local-config", common.DefaultLocalConfigDir, "Local config path")
```

Comment:
Can you please add some more details regarding this flag in the stage1-interface document?